test/jqLiteSpec.js

```diff
@@ -635,6 +635,56 @@ describe('jqLite', function() {
       expect(select.attr('multiple')).toBe('multiple');
     });
 
+    it('should set respective properties when getting/setting boolean attributes', function() {
```

Comment:
I was under the impression that we wouldn't do the patching even if a browser/version failed to do the right thing. If the plan is to do the patching in that case, then I would rather be on the afe side and set the properties anyway. (But if jQuery doesn't do it, I think it is better for jqLite not to do it either.)

I don't feel strongly about keeping the test around, but a different description that makes it clear we expect _the browser_ to do it would probably make things a little easier fr our future selves, if something breaks :smiley: